packages/git-wrapper/tests/GitWorkingCopyTest.php

```diff
@@ -452,6 +452,9 @@ public function testLiveOutput(): void
 
         $this->assertEmpty($empty);
 
+        $gitWrapper->streamOutput(true);
+        $git->status();
```

Comment:
is there possible to create separate test case or use dataProvider true/false for it so existing test is still exists?